lib/qr_code_scanner/example/integration_test/app_test.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2022 Sharezone UG (haftungsbeschränkt)
+// Licensed under the EUPL-1.2-or-later.
+//
+// You may obtain a copy of the Licence at:
+// https://joinup.ec.europa.eu/software/page/eupl
+//
+// SPDX-License-Identifier: EUPL-1.2
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:qr_code_scanner_example/main.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  // Before running the test, add the screenshot "qr_code.jpg" to your Android
+  // emulator camera: https://stackoverflow.com/a/64922184/8358501
+  //
+  // And you need to grand the permission to access the camera.
+  testWidgets(
+    "scan the qr code from the camera",
+    (tester) async {
+      await tester.pumpWidget(const ExampleApp());
+
+      await tester.tap(find.byKey(const Key('scan-qr-code-button-e2e')));
+      await tester.pumpAndSettle();
+
+      await waitFor(tester, find.byKey(const Key('scan-qr-code-button-e2e')));
+
+      expect(
+        find.text('Received QR code: https://sharez.one/pGmfH4rTQeuxXbLE6'),
+        findsOneWidget,
+      );
+    },
+    // Not working at the moment. Sometimes the camera does not include the
+    // screenshot and otherwise you get a 'PlatformException(error, No active
+    // stream to cancel, null, null)' error.
+    skip: true,
```

Comment:
The test should work locally, but could be flaky. Additionally, I'm not able to set it up with GitHub Actions (https://github.com/juliansteenbakker/mobile_scanner/pull/121). Should I delete the code or keep it and skip the test?

Reply:
Hm, I'm not sure. Does the test work locally at all? Do you think there is a chance that we might get such a test to run in the near future? Then we should keep it.  

In any case (even if you delete the test) I think you should create a detailed description on why its not working and what you tried to do to get it to work so no time is wasted in the future if we ever try to add such a test again (e.g. linking to the mentioned `mobile_scanner` issue).

Reply:
Is this still not working? @nilsreichardt